cookbook/case_studies/ml_training/pima_diabetes/Dockerfile

```diff
@@ -29,10 +29,6 @@ RUN ${VENV}/bin/pip install wheel
 COPY pima_diabetes/requirements.txt /root
 RUN ${VENV}/bin/pip install -r /root/requirements.txt
 
-# Copy the makefile targets to expose on the container. This makes it easier to register.
-COPY in_container.mk /root/Makefile
-COPY pima_diabetes/sandbox.config /root
-
 # Copy the actual code
 COPY pima_diabetes/ /root/pima_diabetes/
```

Comment:
@evalsocket: Do we want to change this from `COPY pima_diabetes/ /root/pima_diabetes/` to `COPY . /root/pima_diabetes/`? Similary, `COPY pima_diabetes/requirements.txt /root` to `COPY requirements.txt /root`?

Reply:
^ In accordance with `flytectl` commands.

Reply:
You might also want to change `ghcr_push.yaml` GitHub action, if we incorporate these changes.

Reply:
I think we don't need that change. Can you explain me why it will fail with flytectl?

Reply:
With `flytectl`, I have to serialize and register within `ml_training` directory, correct? For example, here are the commands I have to run for `integrations`: https://github.com/flyteorg/flytesnacks/blob/495eee2fee2fd815ff69c1e41490eb4914052135/cookbook/docs/integrations.rst#deployment. Does this make sense?